GeoLib/IO/FEFLOW/FEFLOWGeoInterface.cpp

```diff
@@ -0,0 +1,209 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "FEFLOWGeoInterface.h"
+
+#include <cctype>
+#include <QtXml>
+
+#include <logog/include/logog.hpp>
+
+#include "BaseLib/FileTools.h"
+#include "BaseLib/StringTools.h"
+
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/Point.h"
+#include "GeoLib/Polygon.h"
+
+
+namespace GeoLib
+{
+namespace IO
+{
+
+void FEFLOWGeoInterface::readFEFLOWFile(const std::string &filename, GeoLib::GEOObjects& geoObjects)
+{
+    std::ifstream in(filename.c_str());
+    if (!in)
+    {
+        ERR("FEFLOWInterface::readFEFLOWFile(): Could not open file %s.", filename.c_str());
+    }
```

Comment:
- Maybe in the error message FEFLOWInterface can be substituted by FEFLOW**Geo**Interface.
- If the stream for reading the data can not be opened you should return here,